drivers/hmis/spec/jobs/merge_clients_job_perf_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe Hmis::MergeClientsJob, type: :model do
```

Comment:
this could go into the other spec for the merge clients job but that's already pretty huge